source/mainnet/smart-contracts/onboarding-guide-solana-developers/overview.rst

```diff
@@ -0,0 +1,112 @@
+.. _solana-overview:
+
+========
+Overview
+========
+
+This overview outlines the main differences of account models and smart contract functionality between Solana and Concordium.
+
+Accounts
+========
+
+Solana accounts are somewhat similar to files and can hold arbitrary data.
+Some accounts are marked as `executable` and contain program code.
+These accounts themselves are immutable, but programs stored in their data can modify data in accounts they own.
```

Comment:
This last part does not sound different from smart contracts on concordium. They are also executable and can modify the data they own.

Reply:
I think it's fine to keep it this way here, it's on Solana side, and this guide is for Solana devs, so they know the details already :)